tests/assets/packages/configurable_hello_world.py

```diff
@@ -1,7 +1,7 @@
-from steamship.app import Invocable, InvocableResponse, create_handler, post
+from steamship.invocable import InvocableResponse, PackageService, create_handler, post
```

Comment:
Would it make sense to move PluginService to this package? That would result in all abstract definitions being bundled under `steamship.invocable`?

Reply:
Sounds good to me. I renamed the file plugin_service.py from service.py as well.